pyat/at/lattice/lattice_object.py

```diff
@@ -144,11 +161,13 @@ def __init__(self, *args,
         kwargs.setdefault('name', '')
         periodicity = kwargs.setdefault('periodicity', 1)
         kwargs.setdefault('_particle', Particle())
+        kwargs.setdefault('_fillpattern', numpy.array([1.0]))
```

Comment:
Should not `_fillpattern` be an array (harm number,) of floats?
Since it looks difficult to set a fill pattern from the constructor keyword arguments, you may just call set_fillpattern() at the end of `__init__`